src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -61,6 +61,50 @@ public void ConstructionFromTimeSpan()
             Offset offset = Snippet.For(Offset.FromTimeSpan(timespan));
             Assert.AreEqual(5400, offset.Seconds);
         }
+        
+        [Test]
+        public void Add()
+        {
+            var leftHandOffset = Offset.FromHours(5);
+            var rightHandOffset = Offset.FromHours(6);
+
+            var result = Snippet.For(Offset.Add(leftHandOffset, rightHandOffset));
+
+            Assert.AreEqual(Offset.FromHours(11), result);
+        }
+
+        [Test]
+        public void Substract()
+        {
+            var leftHandOffset = Offset.FromHours(7);
+            var rightHandOffset = Offset.FromHours(5);
+
+            var result = Snippet.For(Offset.Subtract(leftHandOffset, rightHandOffset));
+
+            Assert.AreEqual(Offset.FromHours(2), result);
+        }
+
+        [Test]
+        public void CompareTo()
+        {
+            var smallerOffset = Offset.FromHours(3);
+            var largerOffset = Offset.FromHours(5);
+
+            var lessThan = Snippet.For(smallerOffset.CompareTo(largerOffset));
+
+            Assert.Less(lessThan, 0);
+        }
+
+        [Test]
+        public void Equals()
+        {
+            var offset1 = Offset.FromHoursAndMinutes(1, 30);
+            var inequalOffset = Offset.FromHours(2);
+
+            var isInequal = Snippet.For(offset1.Equals(inequalOffset));
```

Comment:
Add an equal one as well? (Again, no need for Snippet.For.)

Reply:
ok! Working on it.